### DISCUSSION TITLE
Possible new commands

### DISCUSSION THREAD

**Discussion by YanKnKt** [DiscordDungeons/Bugs#974] [Ideas]
add a command that shows you which commands you can use based in your level.. 
idk if already exists.. but be possible to assign more than 1 attribute in same command, splited by "comma" or ";"..
and if possible a "%prefix%.recipe brew[brewlist]" showing the possibles brew recipes.. ^-^'

**Comment by Feathers8** [comment]
> idk if already exists.. but be possible to assign more than 1 attribute in same command, splited by "comma" or ";"..

Not very useful since the only attributes you'll put your points on are just strength and xpboost lol (and goldboost but that's way later on)

**Comment by Sven65** [comment]
Adding a command that shows which you can use wouldn't be very useful as that information is available on the wiki, and in the `#!help` command.

As for assigning more than one attribute in the same command, it would be a good QoL addition, but I'm unsure how to handle it if a user accidentally assigns to the wrong attribute.

For the brew list, that's not happening, as they aren't intended to be public as players are supposed to discover them by different means (which aren't that many at the moment)
